.env.template

```diff
@@ -0,0 +1,38 @@
+# Example environment configuration file: copy to `.env` (this is git-ignored) and fill in any additional values you need.
+
+# ALGOLIA_APP_ID=
+# ALGOLIA_SEARCH_KEY=
+# ALGOLIA_WRITE_KEY=
+
+# AWS_ACCESS_KEY_ID=
+# AWS_SECRET_ACCESS_KEY=
+
```

Comment:
Not technically needed since it's being fetched via AWS CLI

Reply:
What about some of these other vars? Can we remove any of them? Like, are we using Wiremock for anything? Some of these I've never set like 15-17.

Reply:
Yeah go ahead. Feel free to remove the Wiremock ones (we can remove the related scripts separately) and the Docker desktop password/username (which I don't recall having ever used...) and the Google API key (which shouldn't be needed at this point).